### DISCUSSION TITLE
Can I flash lsposed 1.8.0(zygisk) in TWRP mode? I can not open any app after update to 1.8.1

### DISCUSSION THREAD

**Discussion by suroot66** [LSPosed/LSPosed#1826] [Q&A] [answered]
When I update Lsposed(1.8.0, zygisk) to 1.8.1 and reboot my phone, the system is started and UI is started. But when I unlock the screen, all apps can't be open, including the laucher. I tried to start Thanox from status bar shortcuts, but nothing happen. 
So I go to TWRP and disabled the zygisk-LSPosed, but what worse happened: I can NOT start the system, it keeps stucking at the starting logo.
Now I have went to TWRP, removed the LSP and disabled all magisk modules, but it does not work. I still can boot my system but no app can be started, except system UI and system settings. I don't know what is happening, I've thought magisk and MM for TWRP was absolutely safe, but it seems that I was wrong, lolllll
Now I am thinking about re-flash LSP 1.8.0 through TWRP and see what will happen, so is there a way to do that? Flashing a magisk module through TWRP?  : /

**Comment by yujincheng08** [comment, marked as answer]
No. And 1.8.1 is just some minor bug fix and it won't cause a new bootloop problem/apps crash.

**Reply by suroot66** [reply]
Okay, Thank you for your reply! Maybe I got some trouble not caused by lsposed. Now I uninstalled magisk, and re-flashed my ROM in TWRP, but it still dosen't work. I don't know what happened but there must be something wrong with config data. Maybe I have to clear data and cache...   So sad  T_T
